### DISCUSSION TITLE
Feature request: user menu item like "Your discussions" for issues

### DISCUSSION THREAD

**Discussion by TobiasKnauss** [community/community#27447] [Discussions] [answered]
I was trying to find a discussion or issue where I participated. I found "your discussions" in the user menu, assuming that it would also cover the "discussions" that are part of issue reports. But it does not.
Please either enhance "your discussions" to also show my participation in issue reports, or create another menu item "your issues", which shows all issues and/or comments on issues that I created.
I finally found what I was looking for by entering my user name in the search, but the suggested new feature would make that easier.

**Comment by jfullstackdev** [comment, marked as answer]
is this the one you are referring to ? 

<https://github.com/discussions> 

you can easily find discussions you started or commented

![image](https://user-images.githubusercontent.com/47092464/181505389-c4f5579b-3523-47e9-8eef-92f8398d73ae.png)

for issues, it's the same 

<https://github.com/issues>

![image](https://user-images.githubusercontent.com/47092464/181505482-d4f4a695-3a4c-495a-a5c8-3def81fabbb5.png)
